common/religion/doctrines/00_core_tenets.txt

```diff
@@ -1268,9 +1321,18 @@
 
 	tenet_dharmic_pacifism = { #Mix of reincarnation and pacifism rites
 		icon = core_tenet_dharmawheel
+		
+		# Warcraft
+		name = {
+			first_valid = {
+				desc = tenet_dharmic_pacifism_name_neutral
+			}
+		}
+		
```

Comment:
To replace vanilla localization you need to place the .yml file in `/localization/replace/<language>/`

Reply:
Done in 9fb485782b7c952468bd6db35a94a8aded652f76.